src/WebJobs.Script/Config/TransmissionStatusHandler.cs

```diff
@@ -0,0 +1,147 @@
+﻿//Copyright(c).NET Foundation.All rights reserved.
+//Licensed under the MIT License. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.Json.Serialization;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility.Implementation;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    /// <summary>
+    /// Transmission Status Event Handler
```

Comment:
Do you mind expanding on the documentation here for folks not familiar with that this is. What is a transmission status event handler? How is it used? What is the purpose of this handler etc.

Reply:
Added comment.